site/develop/index.md

```diff
@@ -309,10 +309,9 @@ so it is better to stick with `org.checkerframework.checker.nullness.qual.Nullab
 
 ## Continuous Integration Testing
 
-Calcite exploits [GitHub actions](https://github.com/apache/calcite/actions?query=branch%3Amain)
-and [Travis](https://app.travis-ci.com/github/apache/calcite) for continuous integration testing.
-In the past, there were also Jenkins jobs on the [ASF-hosted](https://builds.apache.org/)
-infrastructure, but they are not maintained anymore.
+Calcite exploits [GitHub actions](https://github.com/apache/calcite/actions?query=branch%3Amain) for continuous
+integration testing. In the past, there were also Jenkins jobs on the [ASF-hosted](https://builds.apache.org/)
+infrastructure and [Travis](https://app.travis-ci.com/github/apache/calcite), but they are not maintained anymore.
```

Comment:
I am not sure we want to keep this historical information around, but if we do we should also mention `AppVeyor` IMO

Reply:
Oh yeah, totally forgot about adding AppVeyor to it after migrating it to GH Actions. I suppose we can just remove that paragraph as it doesn't seem relevant to us.

Reply:
+1 on the removal, we have git history if needed, but I guess nobody will really need this info